charts/launch-agent/values.yaml

```diff
@@ -11,8 +11,34 @@ agent:
       cpu: 1000m
       memory: 1Gi
 
+namespace: wandb-launch
+
 # This should be set to the literal contents of your launch agent config.
-launchConfig: |
+launchConfig:
+   # URL of your W&B server. Defaults to https://api.wandb.ai
+  base_url: https://api.wandb.ai
+
+  # W&B entity (i.e. user or team) name
+  entity: <entity-name>
+
+  # Max number of concurrent runs to perform. -1 = no limit
+  max_jobs: -1
+
+  # List of queues to poll.
+  queues:
+  - default
+
+  # Cloud environment config.
+  environment:
+    type: local #aws|gcp
+
+  # Container registry config.
+  registry:
+    type: local #ecr|gcr
+
+  # Container build config.
+  builder:
+    type: docker #kaniko|noop
```

Comment:
the build should default to noop, I don't think anyone will use the docker builder within k8s

Reply:
we can default to noop but then could you submit a pr to update the docs:
https://docs.wandb.ai/guides/launch/run-agent#agent-configuration

```
The environment, registry, and builder keys are optional. By default, the agent will use no cloud environment, a local docker registry, and a docker builder.
```